src/toplevel.c

```diff
@@ -568,6 +568,22 @@ static jl_module_t *eval_import_from(jl_module_t *m JL_PROPAGATES_ROOT, jl_expr_
     return NULL;
 }
 
+// Format msg and eval `throw(ErrorException(msg)))` in module `m`.
+// Used in `jl_toplevel_eval_flex` instead of `jl_errorf` so that the error
+// location in julia code gets into the backtrace.
+static void jl_eval_errorf(jl_module_t *m, const char* fmt, ...)
+{
+    jl_value_t *throw_ex = (jl_value_t*)jl_exprn(call_sym, 2);
+    JL_GC_PUSH1(&throw_ex);
+    jl_exprargset(throw_ex, 0, jl_symbol("throw"));
```

Comment:
Should use the `jl_builtin_throw` object directly.

Reply:
Thanks, fixed.